README.md

```diff
@@ -125,28 +125,48 @@ which destroys the input `A` but also does not need to allocate as much memory.
 
 These use a very similar syntax to BSplines, with the major exception
 being that one does not get to choose the grid representation (they
-are all `OnGrid`).
+are all `OnGrid`). As such one must specify a set of coordinate arrays
+defining the knots of the array.
+
+In 1D
 ```jl
-A = rand(8,20)
-itp = interpolate(A, Gridded{Linear})
-itp[4,1.2] 
+A = rand(20)
+A_x = collect(1.:40.)
+knots = (a,)
+itp = interpolate(knots,A, Tuple{Gridded{Linear}})
```

Comment:
This `Tuple` shouln't be needed - does it not work for you without it? The minimum modification of the old example I need to get it working, is to add knot vectors:

```jl
A = rand(8,20)
knots = (collect(1:8.), collect(1:.5:10.5))
itp = interpolate(knots, A, Gridded{Linear})
itp[4, 1.2]
```